Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureRequest.h

```diff
@@ -70,6 +70,20 @@ typedef NS_ENUM(NSInteger, BTThreeDSecureShippingMethod) {
     BTThreeDSecureShippingMethodShipToStore
 };
 
+/**
+ The add card challenge request
```

Comment:
Nitpick - let's keep it consistent throughout for "add card" or "card add".

Reply:
Good call - done.